src/cohort-manager.js

```diff
@@ -0,0 +1,132 @@
+const Cohort = require('./cohort')
+const Student = require('./student')
+
+class CohortManager {
+  constructor() {
+    this.cohortArray = []
+    this.studentID = 1
+  }
+
+  addCohort(name, maxCapacity) {
+    if (name.length === 0) {
+      return false
+    }
+    for (let i = 0; i < this.cohortArray.length; i++) {
+      const cohort = this.cohortArray[i]
+      if (name === cohort.name) {
+        return false
+      }
+    }
+    const newCohort = new Cohort(name, maxCapacity)
+    this.cohortArray.push(newCohort)
+    return this.cohortArray
+  }
+
+  removeCohort(name) {
+    for (let i = 0; i < this.cohortArray.length; i++) {
+      const cohort = this.cohortArray[i]
+      if (name === cohort.name) {
+        this.cohortArray.splice(i, 1)
+        return this.cohortArray
+      }
+    }
+    return false
+  }
+
+  searchByCohortName(name) {
+    for (let i = 0; i < this.cohortArray.length; i++) {
```

Comment:
I'm glad you're using for loops instead of iterators such as `.filter()` or `.reduce()`. 
It makes it easier for a beginner like me to read your code and understand what's happening.